### DISCUSSION TITLE
Naming Convention for User Stories in Discussions?

### DISCUSSION THREAD

**Discussion by ybressler** [pair-cg/general-contributions#19] [General]
# How should we name user stories?

Share your thoughts as to how we should do that here!

## 1) Possible solution: `PERSON_NAME`'s User Story –  `PERSPECTIVE / STAKEHOLDER`
How do we feel about this?
Examples:
* Sam's User Story – Executive Director
* Sam's User Story – Executive Director / Audience Members

**Comment by PIARCG** [comment]
WHO YOU ARE/REPRESENTING - Need

**Ex:** EXECUTIVE DIRECTOR - People to see show

**Reply by ybressler** [reply]
Can you give an example of a title so we can see how it works?

**Reply by fjjulien** [reply]
I much prefer @PIARCG's suggestion. It will be much easier to navigate. Moreover, we may eventually merge user stories, in which case the "owner" of a story will become even less relevant.

**Reply by KlausIllmayer** [reply]
I guess it is not necessary to use PERSON_NAME/WHO_YOU_ARE as it is clearly covered by GitHub who is writing an issue/a discussion/etc. (look at the icons and the name of the people in the initial discussion thread) - why not leave it to GitHub? You can even see who has edited a discussion. I guess it would be more necessary to have a clearer understanding, if you aim something for with the naming convention. E.g. if you like to order the user stories in a specific way, you should think about putting the necessary order information on the first place. It looks like that the stakeholders are a fixed list (controlled vocabulary), then it would probably make sense to take the stakeholder in the first place, so you can group the user stories in a very easy way along the different stakeholders.

**Reply by marielmat** [reply]
+1 for @PIARCG proposal, but I guess it might create some duplicates? 

What about a triple-formula following what was presented today - no more than x words for the wantTo/soThat parts : 

_As a [persona], I [want to], [so that]_  =
asA: Archivist ; wantTo: connect collections with Events ; soThat: add context

**Reply by ybressler** [reply]
> +1 for @PIARCG proposal, but I guess it might create some duplicates?
> 
> What about a triple-formula following what was presented today - no more than x words for the wantTo/soThat parts :
> 
> _As a [persona], I [want to], [so that]_ =
> asA: Archivist ; wantTo: connect collections with Events ; soThat: add context

I don't think we'll have an issue with name collisions @mariel-marshall – I also prefer a simpler naming convention to minimize upfront work for creating "something."
